tests/test_deltat.py

```
@@ -43,508 +43,509 @@ def delta_t_long_term(y,m,d):
+    assert dt.first == 1972
+    assert dt.last == _MAX_YEAR
+    assert dt.before == 10
+    assert not dt.is_float
+
+    years = [1971, 1972, 2017, 9999]
+    answer = [10, 10, 37, 37]
+    assert np.all(dt.delta_t_from_ymd(years, 1) == answer)
+    assert np.all(dt.leapsecs_from_ymd(years, 1) == answer)
+
+    # dates out of chronological order
+    with pytest.raises(ValueError):
```

Ruff (0.15.6)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)